CombinatorialGames/Nimber/Simplicity.lean

```diff
@@ -378,6 +373,9 @@ theorem IsRing.one : IsRing 1 where
   mul_lt := by simp
   __ := IsGroup.one
 
+theorem IsRing.of_le_one {x : Nimber} (h : x ≤ 1) : IsRing x := by
+  obtain rfl | rfl := Nimber.le_one_iff.1 h <;> simp
```

Comment:
(Repeated comment) I'm unsure about which one would be better. This pattern also appears again.

```suggestion
  cases Nimber.le_one_iff.1 h <;> simp_all
```